package.json

```diff
@@ -3,26 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/react-hooks": "^3.1.5",
     "@material-ui/core": "^4.10.1",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.55",
     "@material-ui/styles": "^4.10.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "apollo-boost": "^0.4.9",
     "axios": "^0.19.2",
+    "connected-react-router": "^6.8.0",
     "cors": "^2.8.5",
     "create-components-react": "^2.1.4",
+    "graphql": "^15.1.0",
     "react": "^16.13.1",
     "react-customer-chat": "^1.0.2",
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.0",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.1",
+    "react-scripts-config": "^1.0.17-13",
     "react-scroll-up-button": "^1.6.4",
     "react-star-ratings": "^2.3.0",
     "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0",
-    "react-scripts-config": "^1.0.17-13"
+    "redux-saga": "^1.1.3",
+    "redux-thunk": "^2.3.0"
```

Comment:
Please, remove `redux-thunk`, we don't need it anymore

Reply:
Removed redux-thunk dependency